shell/components/Resource/Detail/Card/ResourceUsageCard/composable.ts

```diff
@@ -0,0 +1,51 @@
+import { Props } from '@shell/components/Resource/Detail/Card/ResourceUsageCard/index.vue';
```

Comment:
These are placeholders, I'll be adding the implementation once I work on the cluster dashboard page.